models/pharaoh/pharaoh_avalanche_c_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(
+    schema = 'pharaoh_avalanche_c'
+    , alias = 'trades'
+    , partition_by = ['block_month']
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+    )
+}}
+
+{% set project_start_date = '2023-12-10' %}
+
+WITH dexs AS
+(
+    --QuickSwap v3

```

Comment:
So pharaoh is similar to quickswap v3?

Reply:
That was just in whatever I copied the blurb from. Pharaoh is Ramses which is uni v3.